streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -80,7 +80,11 @@ public ProcessorStateManager(final TaskId taskId,
                                  final boolean eosEnabled) throws IOException {
         this.taskId = taskId;
         this.changelogReader = changelogReader;
-        logPrefix = String.format("task [%s]", taskId);
+        logPrefix = String.format("task [%s] ", taskId);
+
+        final LogContext logContext = new LogContext(logPrefix);
```

Comment:
I think this one should probably be passed in, too. It is related to the `Task`, so makes sense for it to share the same context

Reply:
Agree, will change it too.